Bigtable/src/DataClient.php

```diff
@@ -103,6 +105,15 @@ public function __construct($instanceId, $tableId, array $config = [])
         if (isset($config['bigtableClient'])) {
             $this->bigtableClient = $config['bigtableClient'];
         } else {
+            // Temp workaround for large messages.
+            $config['transportConfig'] = [
```

Comment:
I was testing some large messages and received some error messages about resource limits, this workaround addressed the issue. We can configure it more elegantly when we circle back on how the constructor options will work.